Makefile.js

```diff
@@ -675,17 +674,12 @@ target.gensite = function(prereleaseVersion) {
                 const isFixable = rule && rule.meta.fixable;
                 const hasSuggestions = rule && rule.meta.hasSuggestions;
 
-                // Incorporate the special portion into the documentation content
-                const textSplit = text.split("\n");
-                const ruleHeading = textSplit[0];
-                const ruleDocsContent = textSplit.slice(1).join("\n");
+                text = text.replace("<!-- FIXABLE -->", isFixable ? FIXABLE_TEXT : "")
+                    .replace("<!-- SUGGESTIONS -->", hasSuggestions ? HAS_SUGGESTIONS_TEXT : "")
+                    .replace("<!-- RECOMMENDED -->", isRecommended ? RECOMMENDED_TEXT : "");
 
-                text = `${ruleHeading}${isRecommended ? RECOMMENDED_TEXT : ""}${isFixable ? FIXABLE_TEXT : ""}${hasSuggestions ? HAS_SUGGESTIONS_TEXT : ""}\n${ruleDocsContent}`;
-                title = `${ruleName} - Rules`;
+                title = `${ruleName}`;
```

Comment:
I believe that the `title` variable is unused now and that we can remove it.

Reply:
Right you are.